ted_sws/data_manager/adapters/mapping_suite_repository.py

```diff
@@ -167,6 +168,30 @@ def _write_file_resources(self, file_resources: List[FileResource], path: pathli
             with file_resource_path.open("w", encoding="utf-8") as f:
                 f.write(file_resource.file_content)
 
+    @classmethod
+    def _guess_test_data_file_name(cls, file, parent=None) -> str:
```

Comment:
this looks like some hard-coded magic. I would be wary of it. Certainly there is another way.